spark-cassandra-connector/src/it/scala/com/datastax/spark/connector/rdd/CassandraRDDSpec.scala

```diff
@@ -511,6 +522,43 @@ class CassandraRDDSpec extends SparkCassandraITFlatSpecBase {
     udtValue.getInt("zip") should be(11120)
   }
 
+  it should "allow to save UDT columns from mapped Java objects and read them as UDTValue objects" in {
+    val judt = new JavaTestUDTBean
+    val jb = new JavaTestBean
+
+    // maps to field
+    judt.setField(3)
+    // maps to another_field
+    judt.setAnotherField(4)
+    // maps to yet_another_field
+    judt.setCompletelyUnrelatedField(5)
+
+    // maps to property_1
+    jb.setProperty1(1)
+    // maps to camel_case_property
+    jb.setCamelCaseProperty(2)
+    // maps to nested
+    jb.setNested(judt)
+
+    // We need to bridge the Object mapper in Java to Scala with the JavaBeanColumnMapper
+    implicit val columnMapper = new JavaBeanColumnMapper[JavaTestBean]()
+    sc.parallelize(Seq(jb)).saveToCassandra(ks,"udts_nested")
+
+    // Saving is done via POJO with annotations, now to read them back in
+    val result = sc.cassandraTable(ks, "udts_nested").select("property_1", "camel_case_property", "nested").collect()
+    result should have length 1
+    val row = result.head
+    row.getInt(0) should be(1)
+    row.getInt(1) should be(2)
+
+    val udtValue = row.getUDTValue(2)
+    udtValue.size should be(3)
+    udtValue.getInt("field") should be(3)
+    udtValue.getInt("another_field") should be(4)
+    udtValue.getInt("yet_another_field") should be(5)
+
+  }
```

Comment:
I think we should also enable reading back into the POJO that we just wrote from. Ie on Read we should use the same annotation structure for mapping